incubator/hnc/pkg/controllers/object_controller.go

```diff
@@ -284,14 +292,38 @@ func (r *ObjectReconciler) onDelete(ctx context.Context, log logr.Logger, nnm ty
 	return nil
 }
 
+func (r *ObjectReconciler) ensureCanPropagate(nsNm string, inst *unstructured.Unstructured) bool {
+	r.Forest.Lock()
+	defer r.Forest.Unlock()
+	ns := r.Forest.Get(nsNm)
+	if !ns.Exists() {
+		return false
+	}
+	//TODO TAYLOR
+	var conditionString string
```

Comment:
`String` isn't a very descriptive suffix for a variable of type `string`. Perhaps `conditionKey` is more along the lines of what you meant?